lib/core/test/unit/Cardano/Wallet/DB/Properties.hs

```diff
@@ -761,6 +762,6 @@ newtype GenSparseCheckpointsArgs
 
 instance Arbitrary GenSparseCheckpointsArgs where
     arbitrary = do
-        k <- fromIntegral @Int <$> suchThat arbitrary (>10)
-        h <- fromIntegral @Int . getNonNegative <$> arbitrary
+        k <- (\x -> 10 + (x `mod` 1000)) <$> arbitrary
+        h <- (`mod` 100000) <$> arbitrary
```

Comment:
So, the values generated didn't allow to make the property fail after the initial checkpoint change. We need actually a wider range of values than what the `Int` generator offers us.